python/pen_checker_cdc.py

```diff
@@ -91,18 +92,24 @@ def get_aln_pos_from_ref(hmm_aln,pos):
         ref_pos = ref_pos + 1
     return(ref_pos - 1)
 
-def hmm_search_for_gene(fasta,gene):
+def hmm_search_for_gene(fasta,gene, aa_dir_name, data_dir):
     
     # data structure
     readingframes = []
     
     # from https://www.biostars.org/p/183616/
     # load sequence
+
+    tic_aa_creator = time.perf_counter()
+
     for record in SeqIO.parse(fasta, "fasta"):
-    
-        # process file name
-        gff_base = gff.replace('.gff','')
-        
+
+
+        # process file name expecting no extra .
+        gff_base = os.path.basename(fasta)
+        gff_base = re.sub("\..*[a-z,A-Z,0-9].*$","",gff_base)
+        aa_base_name = aa_dir_name + "/" + gff_base
+
```

Comment:
Tidying up the output name a bit more for readability and to allow for aa sequences to be dropped in new directory where the script is run from